companion/src/logsdialog.cpp

```diff
@@ -399,24 +407,25 @@ void logsDialog::removeAllGraphs()
   ui->customPlot->replot();
 }
 
-void logsDialog::moveLegend()
-{
-  if (QAction* contextAction = qobject_cast<QAction*>(sender())) // make sure this slot is really called by a context menu action, so it carries the data we need
-  {
-    bool ok;
-    int dataInt = contextAction->data().toInt(&ok);
-    if (ok)
-    {
-      ui->customPlot->legend->setPositionStyle((QCPLegend::PositionStyle)dataInt);
-      ui->customPlot->replot();
-    }
-  }
-}
+// void logsDialog::moveLegend()
```

Comment:
Please remove the code if needed, unless it's a TODO and you keep the code for later?